config/roguelike_dungeons/settings/loot_blocks.json

```diff
@@ -0,0 +1,41 @@
+{
+	"name": "loot_blocks",
+	"loot_rules": [
+
+	{"level": [0, 1, 2, 3, 4], "type": "BLOCKS", "each": true, "quantity": 5, "loot": [
+		{"data": {"name": "chisel:hempcrete", "meta": 14, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "minecraft:redstone_lamp", "meta": 0, "min": 8, "max": 24}, "weight": 10},
+		{"data": {"name": "chisel:sveltstone", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "chisel:laboratoryblock", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "HardcoreEnderExpansion:laboratory_glass", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "minecraft:glass", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "minecraft:planks", "meta": 1, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "chisel:woolen_clay", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "chisel:froglight", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "chisel:paperwall_block", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "Botania:prismarine", "meta": 0, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "Botania:prismarine", "meta": 1, "min": 48, "max": 64}, "weight": 10},
+		{"data": {"name": "Botania:prismarine", "meta": 2, "min": 48, "max": 64}, "weight": 10},
```

Comment:
Those Prismarine blocks do have a mana crafting cost associated with them. Consider giving out the full slab instead which doesn't have crafting/recycling recipes.